src/test/ml-modules/root/test/suites/merging-json/merge-option-names-as-json.xqy

```diff
@@ -0,0 +1,22 @@
+xquery version "1.0-ml";
+
+(:
+ : Verify that merge option names are returned correctly.
+ :)
+
+import module namespace test = "http://marklogic.com/roxy/test-helper" at "/test/test-helper.xqy";
+import module namespace lib = "http://marklogic.com/smart-mastering/test" at "/test/suites/merging-json/lib/lib.xqy";
+import module namespace merging = "http://marklogic.com/smart-mastering/merging" at "/com.marklogic.smart-mastering/merging.xqy";
+import module namespace const = "http://marklogic.com/smart-mastering/constants" at "/com.marklogic.smart-mastering/constants.xqy";
+
+
+let $actual := merging:get-option-names($const:FORMAT-JSON)
+
+return (
+  test:assert-same-values(
+    ($lib:OPTIONS-NAME, $lib:OPTIONS-NAME-STRATEGIES, $lib:OPTIONS-NAME-COMPLETE, $lib:OPTIONS-NAME-CUST-XQY,
```

Comment:
I'm getting an error on this test.

Assert Equal failed (ASSERT-EQUAL-FAILED): cust-sjs-test-options cust-xqy-test-options nested-options path-test-options test-options test-options-stock test-options-with-strategies cust-sjs-test-options cust-xqy-test-options json-options json-options-with-strategy nested-options path-test-options test-options test-options-stock test-options-with-strategies